### PR TITLE
Update influxdb.markdown

### PR DESCRIPTION
**Pull request by foxy82**
Added how to query influxdb as the way sensors are stored isn't obvious.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/influxdb.markdown [hunk 1]
```diff
@@ -568,3 +568,34 @@ sensor:
 Note that when working with Flux queries, the resultset is broken into tables, you can see how this works in the Data Explorer of the UI. If you are operating on data created by the InfluxDB history component, this means by default, you will have a table for each entity and each attribute of each entity (other then `unit_of_measurement` and any others you promoted to tags).
 
 This is a lot more tables compared to 1.xx queries, where you essentially had one table per `unit_of_measurement` across all entities. You can still create aggregate metrics across multiple sensors though. As you can see in the example above, a good way to do this is with the [keep](https://v2.docs.influxdata.com/v2.0/reference/flux/stdlib/built-in/transformations/keep/) or [drop](https://v2.docs.influxdata.com/v2.0/reference/flux/stdlib/built-in/transformations/drop/) filters. When you remove key columns Influx merges tables, allowing you to make many tables that share a schema for `_value` into one.
+
+## Querying your data in Infux
```
Comment (klaasnicolaas) [inline source/_integrations/influxdb.markdown]: Infux? 🤣

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/influxdb.markdown (hunk 1 above)
Comment (foxy82) [reply source/_integrations/influxdb.markdown]: 👍 thanks - fixed

File: source/_integrations/influxdb.markdown [hunk 2]
```diff
@@ -568,3 +568,33 @@ sensor:
...
+SELECT * FROM "%" WHERE time > now() - 12h AND "entity_id" = 'multi_sensor_battery_level';
+```
+
+Or for temperatures represented in `°C`:
+
+```sql
+SELECT * FROM "°C" WHERE time > now() - 1h;
+```
+
+### Everythign else
```
Comment (frenck) [inline source/_integrations/influxdb.markdown]:
```suggestion
### Everything else
```

**Review by frenck — APPROVED** [review]
Thanks, @foxy82 👍

**Pull request merged**